docs/modules/reference/pages/how-to-build-with-docker.adoc

```diff
@@ -1,47 +1,47 @@
 
 = Build from Source with Docker
 
-This section describes how to build {page-component-title} with Docker.
+This section describes how to build the {page-component-title} with Docker.
 
 == Simple build
 
-You can compile {page-component-title} from source with a Docker build image:
+You can compile the plugin from source with a Docker build image:
 
 . Check out the repository, compile the code, and install the plugin:
 +
 [source, console]
-docker build -t myopennmshelmimage .
+docker build -t mygrafanapluginimage .
 
 . Run the image:
 +
 [source, console]
-docker run -p 3000:3000 -t myopennmshelmimage
+docker run -p 3000:3000 -t mygrafanapluginimage
 
-. Log in to your {page-component-title} instance with the default admin credentials (`admin`/`admin`).
-. Navigate to *Plugins* and enable the *OpenNMS Helm* plugin.
+. Log in to your Grafana instance with the default admin credentials (`admin`/`admin`).
+. Navigate to *Plugins* and enable the *OpenNMS Plugin for Grafana*.
 
 IMPORTANT: The Grafana plugin directory location is changed from `/var/lib/grafana/plugins` to `/opt/grafana/plugins`.
 This is because the base Grafana image defines `/var/lib/grafana` as a volume, which cannot be changed.
 
 == Advanced build
 
-If you want to build a {page-component-title} image based on a specific GitHub fork or branch, you can use `--build-arg`:
+If you want to build a plugin image based on a specific GitHub fork or branch, you can use `--build-arg`:
```

Comment:
I think in this case, using the full name makes more sense to reduce ambiguity (i.e. to say we're not referring to _a_ plugin, but _our_ plugin).

```suggestion
If you want to build {page-component-title} based on a specific GitHub fork or branch, you can use `--build-arg`:
```